context/app/static/js/components/detailPage/genes/Summary/Summary.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DetailPageSection } from 'js/components/detailPage/style';
+import SummaryBody from 'js/components/detailPage/genes/SummaryBody/SummaryBody';
+
+function Summary() {
+  return (
+    <DetailPageSection id="summary">
+      <SummaryBody />
+    </DetailPageSection>
+  );
+}
+
+Summary.propTypes = {};
+
+Summary.defaultProps = {
+  created_timestamp: undefined,
+  last_modified_timestamp: undefined,
+  description: '',
+  status: '',
+  mapped_data_access_level: '',
+  entityCanBeSaved: true,
+  collectionName: '',
+  mapped_external_group_name: undefined,
+};
```

Comment:
Are these prop types here to support future functionalities in the designs? If some of these are values are going to be passed from flask (e.g. the `variables_with_underscores`), should they be in the flask context instead?